anabeesh/src/main/java/com/cs18/anabeesh/Adapters/DataItem.java

```diff
@@ -0,0 +1,27 @@
+package com.cs18.anabeesh.Adapters;
+
+/**
+ TODO: Add class header
+ */
+
+public class DataItem {
+
+    private final Integer img;
```

Comment:
Leeh Integer?

Reply:
3shan fl Adapter el method   image.setImageResource( "int")  bta5oad  int

Reply:
Change to 
```
@DrwableRes
private final int img;
```